darwin/torch/README.md

```diff
@@ -11,7 +11,7 @@ Here is an example of how to load the `bird-species` dataset ready to be used in
 ```python
 from darwin.torch import get_dataset
 
-db = get_dataset("/datasets/bird-species", dataset_type="instance_segmentation")
+db = get_dataset("/my-datasets/bird-species", dataset_type="instance_segmentation")
```

Comment:
I don't really like the `my-` in `my-datasets`, because of two reasons:
1. The default directory name is `datasets`, so this code won't be copy-pasteable right away for many users
2. A few lines above we wrote `get_dataset("/PATH/TO/YOUR/DATASET", DATASET_TYPE [, PARTITION, SPLIT_TYPE, RELEASE_NAME, TRANSFORMS])`, so it's already clear from the context that the path should be changed if you are not using `datasets`

Reply:
And what about `$DATASETS` instead of `my-datasets`? Because I don't think many users will have a folder called `/datasets` like we do.

Reply:
Sorry, I thought those were cli commands. I changed it to `/datasets/`.